packages/fxa-auth-server/lib/payments/processing-tasks-setup.ts

```diff
@@ -31,12 +33,21 @@ export async function setupProcesingTaskObjects() {
       } as unknown as StatsD);
   Container.set(StatsD, statsd);
 
+  if (!Container.has(AuthFirestore)) {
+    const authFirestore = setupFirestore(config);
+    Container.set(AuthFirestore, authFirestore);
+  }
+
   const log = require('../log')({ ...config.log, statsd });
+  Container.set(AuthLogger, log);
 
   const translator = await require('../senders/translator')(
     config.i18n.supportedLanguages,
     config.i18n.defaultLanguage
   );
+  const profile = require('../profile/client')(log, config, statsd);
+  Container.set(ProfileClient, profile);
```

Comment:
paypal-processor doesn't use this, but I guess some other cron jobs might?

Reply:
Other objects created during the making of the stripe and paypal helper do want this around, yes.